scripts/check_urls.lua

```diff
@@ -0,0 +1,93 @@
+local ts = vim.treesitter
+local async = require('vim._async')
+
+local query = ts.query.parse('vimdoc', '(url) @url')
+
+local function curl(url, cb)
+  local cmd = {
+    'curl',
+    '--silent',
```

Comment:
can `vim.net.request` be used or what is missing from it?

Reply:
the redirect `-L` flag and that the error code value is a string, but I think that checking `err == nil` might be fine too.